pkgs/development/tools/rust/sqlx-cli/default.nix

```diff
@@ -0,0 +1,30 @@
+{ lib, rustPlatform, fetchFromGitHub, pkgconfig, openssl }:
+
+rustPlatform.buildRustPackage {
+  pname = "sqlx-cli";
+  version = "0.2.0";
```

Comment:
```suggestion
  version = "0.4.2";
```

Reply:
I'm confused since the CLI tool has a different version than the libs: https://github.com/launchbadge/sqlx/blob/master/sqlx-cli/Cargo.toml#L3
I think it makes more sense to keep the CLI version.

Reply:
The GitHub tags list 0.4.2 so the version is that. Doing it your way prevents ready updated and everyone thinks it is a mistake.

Reply:
OK it feels weird to have mismatching version, but if you are fine with it then :+1: